adaptive_scheduler/cli.py

```diff
@@ -1,32 +1,35 @@
-#!/usr/bin/env python
 '''
-as.py - Run the adaptive scheduler on continuous loop.
+Run the adaptive scheduler on continuous loop.
 
 Author: Eric Saunders
 July 2012
 '''
 from __future__ import division
 
-from adaptive_scheduler.eventbus import get_eventbus
-from adaptive_scheduler.feedback import UserFeedbackLogger, TimingLogger
-from adaptive_scheduler.interfaces import NetworkInterface
-from adaptive_scheduler.observations import ObservationScheduleInterface
-from adaptive_scheduler.observation_portal_connections import ObservationPortalInterface
-from adaptive_scheduler.configdb_connections import ConfigDBInterface
-from adaptive_scheduler.scheduler import LCOGTNetworkScheduler, SchedulerRunner
-from adaptive_scheduler.scheduler_input import SchedulingInputFactory, SchedulingInputProvider, \
-    FileBasedSchedulingInputProvider, SchedulerParameters
-from adaptive_scheduler.monitoring.network_status import Network
-from adaptive_scheduler.kernel.fullscheduler_ortoolkit import FullScheduler_ortoolkit, ALGORITHMS
+from .eventbus import get_eventbus
```

Comment:
did you need to switch to relative imports? I think I prefer absolute imports, but i guess its just preference nowadays...

Reply:
No didn't have to, just a force of habit. I can change to absolute for consistency, but at some point let's start running our projects through a code formatting tool like https://github.com/google/yapf.